pkg/features/api_client_config.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2020 Red Hat, Inc.
```

Comment:
Not 2020 :)
Istio policy is not to put the year. Anyway, as this is just a move between files, I think it should retain the original value.